src/components/common/container.module.scss

```diff
@@ -18,4 +18,28 @@
   & .container-narrow {
     max-width: 42rem;
   }
+  &.text-heavy {
+    margin: 5em auto;
+    @media (min-width: $viewport-lg) {
+      font-size: 22px;
+      line-height: 1.5;
+    }
+    h2 {
+      font-size: 30px;
+      line-height: 33px;
+      margin: 2rem 0 1rem;
+      padding-top: 1em;
```

Comment:
Two small questions!

1. Do we need the top padding _and_ the top margin? Is this is here for margin collapsing reasons, or could we consolidate the top-edge spacing to the `margin` property?
2. I think this _might_ have introduced a slight layout jog with the permalink on headers? Here’s a screenshot:

![Screen Shot 2020-04-14 at 2 54 30 PM](https://user-images.githubusercontent.com/124493/79275217-c7fff280-7e73-11ea-9ba6-327f006f4386.png)

Reply:
1. Nope -- I collapsed the spacing into `margin` in d274846cd0e1d02c30ee335e167211f5b8d5578c. (Before, we had a vertical space of `69.75px`. Now, we have a vertical space of `72px` using the `4em` margin.)
2. Yes. Fixed!
![image](https://user-images.githubusercontent.com/18607205/79279124-8e76b980-7e6a-11ea-85aa-0565745bc7e2.png)